elasticsearch/sgconfig/sg_roles_mapping.yml

```diff
@@ -2,6 +2,11 @@ sg_role_kibana:
   users:
     - 'CN=system.logging.kibana,OU=OpenShift,O=Logging'
 
+sg_role_prometheus:
+  users:
+    - 'prometheus'
+# ??    - 'system:serviceaccount:{{ openshift_logging_elasticsearch_namespace }}:aggregated-logging-elasticsearch'
```

Comment:
I presume we want this: https://github.com/openshift/openshift-ansible/pull/4509/files#diff-13d085007555088417a56caeb0018db2R55

This also will only work if we either:

* receive the username in the proper forward hearder
* Modify our plugin to set the username based on the bearer token provided